fala/apps/adviser/tests/test_crown_dependencies_playwright.py

```diff
@@ -0,0 +1,24 @@
+from playwright.sync_api import expect
+from fala.playwright.setup import PlaywrightTestSetup
+
+
+class CrownDependenciesTest(PlaywrightTestSetup):
+    def test_jersey(self):
+        results_page = self.visit_results_page("JE1")
+        expect(results_page.h1).to_have_text("The postcode JE1 is in Jersey")
+        search_page = results_page.change_search()
+        # Turns out that Jersey postcodes don't preserve the search term
```

Comment:
yeah I think this must be by design - we don't want them using our service for Jersey. I would reword this comment to that affect. You could check for the absence of the Jersey postcode in the test maybe? So that it is intentional.

Reply:
Thanks, I've update the comment to that effect:

`# We don't preserve Jersey postcodes search term, by design, as we don't want residents from Jersey to use our service`